pkg/frontend/export_test.go

```diff
@@ -162,63 +128,34 @@ func Test_writeToCSVFile(t *testing.T) {
 				Header:   true,
 				FilePath: "test/export.csv",
 			},
-			LineSize: 1,
-			Writer:   &bufio.Writer{},
-			mrs:      &MysqlResultSet{},
+			mrs: &MysqlResultSet{},
 		}
 
 		var output = []byte{'1', '2'}
 		ep.userConfig.MaxFileSize = 1
 
-		convey.So(writeToCSVFile(ep, output), convey.ShouldNotBeNil)
-
-		ep.Rows = 1
-		stubs := gostub.StubFunc(&Flush, moerr.NewInternalError(context.TODO(), "Flush error"))
```

Comment:
Means, these variables has beend removed in the PR.

Reply:
yes, Writer is removed.  We only support AsyncWriter and AsyncReader.  For LineSize, it is same as FileSize. variables should be removed.